src/olympia/amo/monitors.py

```diff
@@ -31,6 +31,28 @@ def execute_checks(checks: list[str]):
     return status_summary
 
 
+def web():
```

Comment:
As discussed on slack let's rename that to `localdev_web` to be less confusing

Reply:
Done.